guide/packages.md

```diff
@@ -107,14 +120,24 @@ The Membrane docker ([Docker Hub](https://hub.docker.com/r/membrane/membrane), [
+| `membrane_video_compositor_plugin`         | [Alpha] WGPU based plugin for composing, transforming and editing multiple raw videos in real time | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework-labs/membrane_video_compositor_plugin)                     |
+| `membrane_video_mixer_plugin`        | [Experimental] [Third-Party] Element which mixes multiple video inputs to a single output using ffmpeg filters | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_video_mixer_plugin.svg)](https://hex.pm/packages/membrane_video_mixer_plugin) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/kim-company/membrane_video_mixer_plugin)                     |
+
+### RTC
+
+| Package                                  | Description                                                              | Links                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| ---------------------------------------- | ------------------------------------------------------------------------          | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `membrane_rtc_engine`                    | RTC Engine and its client library                                                 | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_rtc_engine.svg)](https://hex.pm/packages/membrane_rtc_engine) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_rtc_engine) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_rtc_engine)             |
+| `membrane_webrtc_plugin`                 | [Alpha] Plugin for sending and receiving media with WebRTC                        | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_webrtc_plugin.svg)](https://hex.pm/packages/membrane_webrtc_plugin) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_webrtc_plugin) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_webrtc_plugin) |
```

Comment:
Shouldn't it be put in webrtc/libraries?

Reply:
I was considering that too, but both of them provide bins / elements